runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -372,7 +372,15 @@ else if (braceletText.contains("expeditious bracelet"))
 				Matcher mPoints = REWARD_POINTS.matcher(w.getText());
 				if (mPoints.find())
 				{
+					int prevPoints = points;
```

Comment:
This can be `final` since it's only assigned once.
```suggestion
					final int prevPoints = points;
```